vllm/attention/layer.py

```diff
@@ -575,6 +575,7 @@ def unified_attention_fake(
     mutates_args=[],
     fake_impl=unified_attention_fake,
     dispatch_key=current_platform.dispatch_key,
+    tags=(torch._C.Tag.cudagraph_unsafe, ),
```

Comment:
@BoyuanFeng , @ProExpertProg , may you help to make this conditional to cuda-alike device? It failed HPU, I think CPU and XPU should also gets impacted?

Reply:
What's the failure? Feel free to create a PR

Reply:
Thanks, I submitted a PR : https://github.com/vllm-project/vllm/pull/25298

Reply:
Thanks for fixing!